op-e2e/system_adminrpc_test.go

```diff
@@ -172,6 +177,54 @@ func TestLoadSequencerStateOnStarted_Started(t *testing.T) {
 	assertPersistedSequencerState(t, stateFile, node.StateStarted)
 }
 
+func TestPostUnsafePayload(t *testing.T) {
+	InitParallel(t)
+	ctx := context.Background()
+
+	cfg := DefaultSystemConfig(t)
+	cfg.Nodes["verifier"].RPC.EnableAdmin = true
+
+	sys, err := cfg.Start(t)
+	require.NoError(t, err)
+	err = sys.BatchSubmitter.Stop(ctx)
+	require.NoError(t, err)
+	defer sys.Close()
+
+	l2Seq := sys.Clients["sequencer"]
+	l2Ver := sys.Clients["verifier"]
+	rollupRPCClient, err := rpc.DialContext(ctx, sys.RollupNodes["verifier"].HTTPEndpoint())
+	require.Nil(t, err)
+	rollupClient := sources.NewRollupClient(client.NewBaseRPCClient(rollupRPCClient))
+
+	time.Sleep(time.Duration(cfg.DeployConfig.L2BlockTime+1) * time.Second)
+	seqBlock, err := l2Seq.BlockByNumber(ctx, nil)
+	require.NoError(t, err)
+	require.Greater(t, seqBlock.Number().Uint64(), uint64(0), "Sequencer should have advanced at least one block")
```

Comment:
We shouldn't use a fixed sleep period - there's no guarantee that async actions will actually happen in time, particularly on busy CI boxes.  Instead we should explicitly wait for the action we're interested in. In this case I think you can just use `require.NoError(t, wait.ForBlock(ctx, l2Seq, 1))` to wait for there to be at least one block. 

```suggestion
	require.NoError(t, wait.ForBlock(ctx, l2Seq, 1))
```